src/collections/_documentation/platforms/android/index.md

```diff
@@ -0,0 +1,597 @@
+```
+
+### Setting the Fingerprint
+
+Sentry uses a fingerprint to decide how to group errors into issues.
+
+For some very advanced use cases, you can override the Sentry default grouping using the `fingerprint` attribute. In supported SDKs, this attribute can be passed with the event information and should be a list of strings.
+
+For code samples, see [Grouping & Fingerprints]({%- link _documentation/data-management/event-grouping/index.md -%}#use-cases).
+
+For more information, see [Aggregate Errors with Custom Fingerprints](https://blog.sentry.io/2018/01/18/setting-up-custom-fingerprints).
+
+### Default Context for Android
+
+To make your life easier, the Android SDK automatically enriches the context with additional information about the device, about the system, and about the release version of your application.
+
+Default Android context should help you correlate the errors with the specific device types, releases, etc. so that you can find the root cause of the problem faster.
+
+Used SDK 
+
+- name
+- version
+
+Operating system 
+
+- name
+- version
+- kernel version
+- whether the application was rooted
+
+Package info
+
+- Start time of the application
+- Bundle name
+- bundle id
+- version
+- build
+
+Memory size 
+
+- Total available memory on the device
+- Free memory at the moment
+- Whether the memory was low
+
+Storage
+
+- total storage size
+- free storage size
+- total external storage size
+- free external storage size
+
+Device Info
+
+- Manufacturer
+- Brand
+- Model
+- Model ID
+- Architecture
+
+Battery Level
+
+- Whether the Device is being charged at the moment
+- Whether the phone was online during the error/crash
+
+Screen info
+
+- orientation of the screen
+- screen density
+- hight pixels
+- width pixels
+- resolution
+
+Emulator indicator
+
+- Whether the device is an emulator
+
+## Advanced Usage
+
+### Requirements
+
+For the use of the SDK with the NDK, the minimal required API level is 21. 
+
+If you want to use the SDK without the NDK, you can: 
+
+- disable the NDK as described in the configuration section
+- or use sentry-android-core that doesn't contain the NDK and can be used separately. The minimal required API level for this SDK is 14.
+
+To add the sentry-android-core library, you need to provide the following dependency in your build.gradle file.
+
+```groovy
+// ADD COMPATIBILITY OPTIONS TO BE COMPATIBLE WITH JAVA 1.8
+compileOptions {
+        sourceCompatibility = JavaVersion.VERSION_1_8
+        targetCompatibility = JavaVersion.VERSION_1_8
+    }
+
+// ADD SENTRY ANDROID AS A DEPENDENCY
+dependencies {
+        implementation 'io.sentry:sentry-android-core::2.0.+'
+}
+```
+
+If you want to use the SDK with the NDK but you still want to support the devices on the API level lower than 21 you can update your manifest as follows:
+
+```xml
+    <!-- Merging strategy for the imported manifests -->
+      <uses-sdk
+            tools:overrideLibrary="io.sentry.android, io.sentry.android.ndk"/>
```

Comment:
`tools:overrideLibrary="io.sentry.android"/>` only, I've removed the necessary of `io.sentry.android.ndk`, will be available on version alpha08